DOCS-2567 iOS Crash Reporting and Error Tracking Edits (#776)
DataDog/dd-sdk-ios: docs/rum_collection/crash_reporting.md

Every iOS application produces dYSM files for each application module. Each application version contains a set of dYSM files.

Depending on your setup, you may need to download dSYM files from App Store Connect or find them on your local machine. 

| Bitcode Enabled | Description                                                                                                                                                                                                                                                                                       |
|-----------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| Yes             | dSYM files are available once [App Store Connect][6] completes processing your application's build.                                                                                                                                                                                                    |

> | No              | Xcode exports dSYM files to `$DWARF_DSYM_FOLDER_PATH` at the end of your application's build. Ensure that the `DEBUG_INFORMATION_FORMAT` build setting is set to **DWARF with dYSM File**. By default, Xcode projects only set `DEBUG_INFORMATION_FORMAT` to **DWARF with dSYM File** in release. |

Reviewer: ```suggestion
| No              | Xcode exports dSYM files to `$DWARF_DSYM_FOLDER_PATH` at the end of your application's build. Ensure that the `DEBUG_INFORMATION_FORMAT` build setting is set to **DWARF with dYSM File**. By default, Xcode projects only set `DEBUG_INFORMATION_FORMAT` to **DWARF with dSYM File** for _Release_ configuration. |
```
Where "Release" is the name of project configuration (like below), so I believe it should be upper cased.

<img width="521" alt="Screenshot 2022-03-14 at 17 49 02" src="https://user-images.githubusercontent.com/2358722/158220829-3a8b83a3-15b7-49bb-9769-27c8d916b3a8.png">

### Upload your dYSM file

> dYSM files minimize an application's binary size and enable faster download speed. By uploading your dYSM file to Datadog, you gain access to the file path, line number, and code snippet of each frame in an error's related stack trace.

Reviewer: ```suggestion
By uploading your dYSM file to Datadog, you gain access to the file path, line number, and code snippet of each frame in an error's related stack trace.
```

nit: It feels out of the context in this instruction. I think this sentence fits perfectly what we say above in "Find your dYSM file", WDYT?
```diff
### Find your dYSM file

Every iOS application produces dYSM files for each application module. 
+ These files minimize an application's binary size and enable faster download speed.
Each application version contains a set of dYSM files.
```

            path/to/zip/dsyms.zip
```

For more information, see [dSYMs commands][7].

## Verify crash reports

To verify your iOS Crash Reporting and Error Tracking configuration, issue a crash in your RUM application and confirm that the error appears in Datadog. 

1. Run your application on an iOS simulator or a real device. Ensure that the debugger is not attached. Otherwise, Xcode captures the crash before the iOS SDK does.
2. Execute the code containing the crash:

>    ```
>    func didTapButton() {
>    fatalError(“Crash the app”)
>    }
>    ```

Reviewer: ```suggestion
   ```swift
   func didTapButton() {
      fatalError(“Crash the app”)
   }
   ```
```
